fvtr/python/python.exp

```diff
@@ -105,6 +105,10 @@ set env(TMPDIR) "$FULLPATH/tmp[pid]"
 exec mkdir $env(TMPDIR)
 printit "TMPDIR=\"$env(TMPDIR)\""
 
+# Test for the issue #45
+printit "Executing sqlite3 importing test."
+exec $env(AT_DEST)/bin/python3 -c "import sqlite3"
+
```

Comment:
In your previous pull request related to this patch, I made a suggestion and a question.

You did answer my question, correcting a wrong assumption from my part.
However, there were no mentions about the suggestion to reuse the test test_sqlite provided by the Python testsuite.
Don't you think this test would have a better coverage than just importing the sqlite3 module?

Notice that what I'm proposing here is to behave the same way as the other tests, e.g. test_gdb, test_site, test_httpservers, test_os, test_posix, test_compileall, etc.